src/Compiler/.editorconfig

```diff
@@ -86,7 +86,7 @@ dotnet_diagnostic.CA1018.severity = warning
 dotnet_diagnostic.CA1047.severity = warning
 
 # CA1305: Specify IFormatProvider
-dotnet_diagnostic.CA1305.severity = warning
+dotnet_diagnostic.CA1305.severity = none
```

Comment:
I guess this is unnecessary since it's already set in the root editorconfig? But anyways, don't we want to have it enabled? At least eventually.

Reply:
It's necessary though because this `.editorconfig` lists itself as a root so we don't actually consider the one above for files below.